client/post-editor/editor-seo-accordion/index.jsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Accordion from 'components/accordion';
+import AccordionSection from 'components/accordion/section';
+import CountedTextarea from 'components/forms/counted-textarea';
+import Gridicon from 'components/gridicon';
+import InfoPopover from 'components/info-popover';
+import TokenField from 'components/token-field';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteSlug } from 'state/sites/selectors';
+
+class EditorSeoAccordion extends Component {
```

Comment:
Not sure how you feel about them, but this looks a perfect candidate for [stateless function component](https://facebook.github.io/react/docs/reusable-components.html#stateless-functions).